programs/mango-v4/src/accounts_ix/openbook_v2_edit_market.rs

```diff
@@ -0,0 +1,22 @@
+use anchor_lang::prelude::*;
+
+use crate::state::*;
+
+#[derive(Accounts)]
+#[instruction(market_index: OpenbookV2MarketIndex)]
+pub struct OpenbookV2EditMarket<'info> {
+    #[account(
+        // group <-> admin relation is checked at #1
+        constraint = group.load()?.openbook_v2_supported(),
+        constraint = group.load()?.admin == admin.key(),
```

Comment:
the comment doesn't make sense, since the admin pubkey is already checked here

Reply:
Removed the comment.